packages/react-query/src/utils/structuralSharing.ts

```diff
@@ -0,0 +1,83 @@
+import equal from 'fast-deep-equal/es6';
```

Comment:
We don't like to have dependencies, so please remove this

Reply:
Done, I copy-pasted the code.